projects/plugins/jetpack/modules/masterbar/admin-menu/class-wpcom-admin-menu.php

```diff
@@ -323,6 +323,13 @@ public function add_appearance_menu() {
 
 		$user_can_customize = current_user_can( 'customize' );
 
+		if ( function_exists( 'wp_is_block_theme' ) && wp_is_block_theme() ) {
+			add_filter( 'safecss_is_freetrial', '__return_false', PHP_INT_MAX );
+			if ( class_exists( 'Jetpack_Custom_CSS' ) && empty( Jetpack_Custom_CSS::get_css() ) ) {
+				$user_can_customize = false;
+			}
```

Comment:
I think it'd be safer to remove the filter afterwards just in case it's used later in the process.
```suggestion
			add_filter( 'safecss_is_freetrial', '__return_false', PHP_INT_MAX );
			if ( class_exists( 'Jetpack_Custom_CSS' ) && empty( Jetpack_Custom_CSS::get_css() ) ) {
				$user_can_customize = false;
			}
			remove_filter( 'safecss_is_freetrial', '__return_false', PHP_INT_MAX );
```